spec/board_spec.rb

```diff
@@ -5,14 +5,19 @@
 
   it "sets token at empty coordinate" do
     expect(board.get_token_at(7)).to eq('7')
-    board.put_token_in_board([7, -1], 'X')
+
+    board.set_token_at(7, 'X')
+
     expect(board.get_token_at(7)).to eq('X')
   end
 
   it "resets token at occupied coordinate" do
-    board.put_token_in_board([7, -1], 'X')
+    board.set_token_at(7, 'X')
+
     expect(board.get_token_at(7)).to eq('X')
+
     board.reset_token_at(7)
+
     expect(board.get_token_at(7)).to eq('7')
   end
```

Comment:
agregaste comportamiento a Board pero no pruebas (rows, cols, diagonals)
eso lo pruebas a través de Referee? (en ese caso me parece bien pq solo son importantes para referee)

Reply:
That's right, I test this cases in the referee.